src/components/Activities/Activity1/index.tsx

```diff
@@ -92,6 +95,41 @@ function Activity1(): JSX.Element {
       topText: 'Computers have to learn just like we do: by trial and error!',
       child: <Slide3 />,
     },
+    {
+      child:
+        <div id="game2-intro">
+          <span>But even if we know what the alien is saying...can you figure out what they mean?</span>
+          <br/>
+          <span>Warning: One sentence can mean two things, so the answer might not be what you expect!</span>
+          <br/>
+          <button id="game2-intro-button" onClick={context.next}>
+            Play Game
+          </button>
+        </div>,
+    },
+    {
+      child: <AmbiguousGame />,
+      showNext: false,
+    },
+    {
+      child:
+        <div>
+          <h2 id={'body-text'}> Being a computer sure isn&apos;t easy... next time you use a search bar, now you know what it has to deal with! </h2>
+
+          <div id='blank-computer'>
+            <img id='blank-computer-img' src={BlankComputer} />
+            <div id='blank-computer-text'>
+              <div id='blank-computer-title-text'> More Information</div> <br/>
+              Computers in the real world use artificial intelligence (AI) to remember what they learn
+              from trial and error. They can share what they learn with other computers in order to
+              give us a better searching experience.
+              <br/> <br/>
+              When you&apos;re older, you&apos;ll get the chance to learn how to code you that you can
+              learn how AI works in more detail!
+            </div>
+          </div>
+        </div>,
+    },
   ];
 
```

Comment:
something like this is probably what you want so that when `AmbiguousPhrasingGameIntro` is rendered, it is rendered within `Carousel`, so the `CarouselContext` will be passed properly!
```suggestion
  const AmbiguousPhrasingGameIntro = () => {
    const context = useContext(CarouselContext);
    return (
      <div id="game2-intro">
          <span>But even if we know what the alien is saying...can you figure out what they mean?</span>
          <br/>
          <span>Warning: One sentence can mean two things, so the answer might not be what you expect!</span>
          <br/>
          <button id="game2-intro-button" onClick={context.next}>
            Play Game
          </button>
        </div>
    );
  };
```